src/Config/TypeWithOutputFieldsDefinition.php

```diff
@@ -16,9 +16,6 @@
     protected function outputFieldsSelection(string $name = 'fields')
     {
         $node = self::createNode($name);
-        $node
```

Comment:
this should not be removed since builder is process in a normalizer before this part

Reply:
You are right, my mistake, sorry. I put it back and removed the useless code I added to check the fields requirements